codice/ddf
# DDF-5996 Simplifies the Web Context Policy Manager's authentication type configuration

#### What does this PR do?
Simplifies the Web Context Policy Manager's authentication type configuration by:
- Removing the authentication type configuration per context
- Adding a single auth type config for all web pages  
- Adding another auth type config for all endpoints
- Keeping the option to configure multiple auth types per context as a file-based configuration

#### Who is reviewing it? 
@bakejeyner
@garrettfreibott  
@ricklarsen 
@SmithJosh 
@stustison 

#### Select relevant component teams: 
@codice/security 

#### How should this be tested?
- Test that changes to the web and endpoint authentication types works as expected
- Test the current (soon to be old) was of configuring auth types is still supported and works
To see how to do these and test the documentation see the docs on how to accomplish ☝️ 

#### What are the relevant tickets?
Fixes: #5996 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
